src/cmdstan-guide/pathfinder_intro.Rmd

```diff
@@ -0,0 +1,129 @@
+# Variational Inference using Pathfinder
+
+The CmdStan method `pathfinder` uses the Pathfinder algorithm of @zhang_pathfinder:2022.
+Pathfinder is a variational method for approximately
+sampling from differentiable log densities. Starting from a random
+initialization, Pathfinder locates normal approximations to the target
+density along a quasi-Newton optimization path, with local covariance
+estimated using the inverse Hessian estimates produced by the
+L-BFGS optimizer.  Pathfinder returns draws from the Gaussian approximation
+with the lowest estimated Kullback-Leibler (KL) divergence to the true
+posterior.
+
+Pathfinder differs from the ADVI method in that it uses quasi-Newton
+optimization on the log posterior instead of stochastic gradient descent (SGD).
+The L-BFGS algorithm is both faster and more stable than SGD.
+Compared to ADVI and short dynamic HMC runs, Pathfinder
+requires one to two orders of magnitude fewer log density and gradient
+evaluations, with greater reductions for more challenging posteriors.
+
+A single run of the Pathfinder algorithm generates a set of approximate draws.
+Inference is improved by running multiple Pathfinder runs and using
+Pareto-smoothed importance resampling (PSIS) of the resulting sets of draws.
+This better matches non-normal target densities and also eliminates minor modes.
+By default, the `pathfinder` method uses 4 independent Pathfinder runs,
+each of which produces 1000 approximate draws.
```

Comment:
I think you should continue after "draws" with ", which are then importance resampled down to 100 final draws."

Reply:
you mean 1000, right?  changed.